### PR TITLE
EE10 and MP6 GA

### LINKED ISSUE #24165: Remove ibm:beta for servlet metatype
Remove beta metatype upon EE10 GA

```
<AD id="skipEncodedCharVerification"
            name="%skipEncodedCharVerification.name"
            description="%skipEncodedCharVerification.desc"
            ibm:beta="true"
            required="false" type="Boolean" default="false" />
```

### PR DESCRIPTION
**Pull request by tevans78**
Move EE10 and MP6 to GA

Fixes #24165

### CODE REVIEW

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
Combined PB: https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_3AACsGdWEe2FXtyvmy2tqA

PB was green and also passed initial Liberty repo tests

File: dev/com.ibm.websphere.appserver.features/visibility/public/restfulWSLogging-3.0/io.openliberty.restfulWSLogging-3.0.feature [hunk 1]
```diff
@@ -9,6 +9,6 @@ Subsystem-Name: Jakarta RESTful Web Services 3.0 with HTTP logging
 -features=io.openliberty.restfulWS-3.0; ibm.tolerates:="3.1"
 -bundles=\
  io.openliberty.restfulWS.internal.logging.filter
-kind=beta
+kind=ga
```
Comment (jhanders34) [inline dev/com.ibm.websphere.appserver.features/visibility/public/restfulWSLogging-3.0/io.openliberty.restfulWSLogging-3.0.feature]: This should not go to GA.

**Review by jhanders34 — CHANGES_REQUESTED** [review]

**Comment by jhanders34** [comment]
I also wonder about the supersedes on whether that is completely right to be all of the mpOpenTracing and opentracing features to have supersede in them.  You cannot use mpTelemetry with EE7, 8 or 9 or MP 1 through 5.  It is a documentation only thing I believe or possibly puts out a message in the messages.log.

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/restfulWSLogging-3.0/io.openliberty.restfulWSLogging-3.0.feature (hunk 1 above)
Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/public/restfulWSLogging-3.0/io.openliberty.restfulWSLogging-3.0.feature]: removed

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
Combined PB: https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_fmLpoLaUEe2r6re_Zvbagg

**Comment by tevans78** [comment]
Another combined PB: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_nB8o0LdeEe2r6re_Zvbagg

**Comment by tevans78** [comment]
combined PB: https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_bdhSkLg-Ee2r6re_Zvbagg

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
Combined PB: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_RRUKALhOEe2r6re_Zvbagg

File: dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/config/liberty/CDI12ContainerConfig.java [hunk 2]
```diff
@@ -48,8 +47,7 @@ protected void activate(ComponentContext compcontext, Map<String, Object> proper
         if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
             Tr.debug(tc, "Activating " + this);
         }
-        //TOOD the ProductInfo.getBetaEdition() check should be removed before GA
-        if (TraceComponent.isAnyTracingEnabled() && tc.isInfoEnabled() && ProductInfo.getBetaEdition()) {
+        if (TraceComponent.isAnyTracingEnabled() && tc.isInfoEnabled()) {
```
Comment (jhanders34) [inline dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/config/liberty/CDI12ContainerConfig.java]: @tevans78 is on vacation.  He put this check in.  Not sure if this is what was intended or not.  Do we want this message to come out every time or only when the customer enables some sort of fine, finer or finest trace?  If the former the isAnyTracingEnabled should be removed.  @Emily-Jiang or @benjamin-confino do you know about what the conversations where related to this change?

**Comment by jhanders34** [comment]
#build

File: dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/config/liberty/CDI12ContainerConfig.java (hunk 2 above)
Comment (Emily-Jiang) [reply dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/config/liberty/CDI12ContainerConfig.java]: I am not sure. @Azquelt do you know anything about this?

**Comment by jhanders34** [comment]
#libby #build

File: dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/config/liberty/CDI12ContainerConfig.java (hunk 2 above)
Comment (Azquelt) [reply dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/config/liberty/CDI12ContainerConfig.java]: Good spot @jhanders34, this looks like a bug. The message should be output whether tracing is enabled or not. The intention is thankfully listed [in the UFO](https://ibm.ent.box.com/file/925747728313) (slide 45).

It looks like the test which checks this message doesn't intentionally set any trace specification, but when it runs it has `*=info:com.ibm.ws.runtime.update.*=all:logservice=detail` which I guess is added automatically.

Comment (jhanders34) [reply dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/config/liberty/CDI12ContainerConfig.java]: Thanks Andrew.  I have updated the code to remove the if statement completely.

**Review by helyarp — COMMENTED** [review]
Looks good

**Comment by posmith** [comment]
L2 message review not required

**Comment by pmd1nh** [comment]
reviewed dev/com.ibm.ws.webcontainer/resources/OSGI-INF/metatype/metatype.xml

**Review by pmd1nh — DISMISSED** [review]
dev/com.ibm.ws.webcontainer/resources/OSGI-INF/metatype/metatype.xml

**Review by jhanders34 — DISMISSED** [review]

**Comment by jhanders34** [comment]
#libby

**Review by teddyjtorres — DISMISSED** [review]
MP JWT 2.1 and Jakarta Security 3.0 changes look good.

**Review by Channyboy — DISMISSED** [review]

**Review by cbridgha — APPROVED** [review]
Looks good

**Comment by jhanders34** [comment]
#libby

**Pull request merged**